### ISSUE TITLE
Optional leading article for constant doc comments.

### ISSUE THREAD

**Issue by mewmew** [golang/lint#11]
Would it be possible to relax the comment requirements for constants so they can also include an optional leading article.

Below is a simple example:

``` go
package p

// Mod corresponds to a bitfield of modifier key flags.
type Mod int

// Modifier key flags.
const (
    // The ModControl bit is set if one or more Control keys were held down.
    ModControl Mod = 1 << iota
    // The ModShift bit is set if one or more Shift keys were held down.
    ModShift
    // The ModAlt bit is set if one or more Alt keys were held down.
    ModAlt
    // The ModSuper bit is set if one or more Super keys were held down.
    ModSuper
)
```

Running golint on `p` gives the following warnings:

```
p/p.go:8:2: comment on exported const ModControl should be of the form "ModControl ..."
p/p.go:10:2: comment on exported const ModShift should be of the form "ModShift ..."
p/p.go:12:2: comment on exported const ModAlt should be of the form "ModAlt ..."
p/p.go:14:2: comment on exported const ModSuper should be of the form "ModSuper ..."
```
